src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -111,11 +112,18 @@ public async Task<AmlResult> PerformAmlCheckAsync(string appId, AsymmetricCipher
             if (string.IsNullOrEmpty(authDigest))
                 throw new InvalidOperationException("Could not sign request");
 
+            string SDKversion = typeof(YotiClientEngine).GetTypeInfo()?.Assembly?.GetName()?.Version?.ToString();
```

Comment:
The variable name should be in camelCase `SDKVersion`?

Reply:
Good spot